### PR TITLE
AP-348 Publish 'Prepare your users to move to GOV.UK One Login' on product site

### PR DESCRIPTION
**Pull request by pauldougan**
[AP-348](https://govukverify.atlassian.net/browse/AP-348)

# Onboarding Feature Deployment

> [!WARNING]
> Pull requests merged to `main` will be released to production, please ensure the checklist below is complete

Before any work can be merged to main in must meet the definition of done and be ready to deploy. While many of these tasks will be automated, the reviewers must take the responsibility of confirming the checklist below has been completed before this ticket can be merged.

## Checklist

-   [x] this pull request meets the acceptance criteria of the ticket
-   [x] this branch is up-to-date with the main branch

    `git fetch --all && git rebase origin/main`

-   [x] these changes are backwards compatible (no breaking changes)

-   all methods signatures and return values are the same
-   any replaced methods are marked as `@deprecated`

## Changes 

Add new guidance to the design recommendations https://www.sign-in.service.gov.uk/documentation/design-recommendations/how-to-talk-about

- add nunjucks template
- add new content to side bar
- add new router rules for the content

## How to test
- `npm install && npm run local` 
- go to https://localhost:3000/documentation/design-recommendations/how-to-talk-about

[AP-348]: https://govukverify.atlassian.net/browse/AP-348?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by pauldougan** [comment]
@CarlyG55 any chance someone from the team can approve this simple PR that adds some static copy to the product pages for our content designer?

File: src/routes/site.ts [hunk 1]
```diff
@@ -123,6 +123,10 @@ router.get("/documentation/design-recommendations/how-to-talk-about", (req, res)
     res.render("documentation-how-to-talk-about.njk");
 });
 
+router.get("/documentation/design-recommendations/prepare-move-content", (req, res) => {
```
Comment (Ryan-Andrews99) [inline src/routes/site.ts:126]: Tiny comment, could we make the url something that doesn't include the word content? Doesn't feel like it fits massively with the standards here: https://www.gov.uk/guidance/content-design/url-standards-for-gov-uk

Comment (pauldougan) [reply src/routes/site.ts:126]: hi Ryan I spoke to Tracy our content dev and agreed to rename the offending file to `documentation-prepare-to-move.njk` I will make a commit after standup

Comment (pauldougan) [reply src/routes/site.ts]: @Ryan-Andrews99 i renamed the file so the pr is ready to approve now

**Review by Ryan-Andrews99 — APPROVED** [review]

**Pull request merged**